pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -130,16 +147,12 @@ func validateThresholds(thresholds api.ResourceThresholds) bool {
 	return true
 }
 
-//This function could be merged into above once we are clear.
-func validateTargetThresholds(targetThresholds api.ResourceThresholds) bool {
-	if targetThresholds == nil {
-		klog.V(1).Infof("no target resource threshold is configured")
-		return false
-	} else if _, ok := targetThresholds[v1.ResourcePods]; !ok {
-		klog.V(1).Infof("no target resource threshold for pods is configured")
-		return false
+// validateResourcePercentage checks if resource percentage is in the range [0, 100]
+func validateResourcePercentage(percent api.Percentage) bool {
```

Comment:
I don't think `validateResourcePercentage` is the right function to unit test given its length and complexity. Can you move the distinct cases into TestValidateThresholds and drop `TestValidateResourcePercentage` completely?

Reply:
done